mixer/template/apikey/template.proto

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package apiKey;
+
+import "google/protobuf/timestamp.proto";
+import "mixer/v1/template/extensions.proto";
+
+option (istio.mixer.v1.template.template_variety) = TEMPLATE_VARIETY_CHECK;
+
+// Template to check if an API call should proceed.
+message Template {
+    // The API being called (api.service).
+    string api = 1;
+
+    // The version of the API (api.version).
+    string api_version = 2;
+
+    // The API operation is being called.
+    string api_operation = 3;
+
+    // API key used in API call.
+    string api_key = 4;
+
+    // Timestamp of API call.
+    google.protobuf.Timestamp timestamp = 5;
```

Comment:
should we name this `request_time` or `operation_time` or `call_time` to match the style of the standard fields in https://cloud.google.com/apis/design/standard_fields ?

Reply:
IMHO, in this context, it doesn't appear to be absolutely necessary to be as precise as call_time, just like the timestamp field in logentry template.